types/node/events.d.ts

```diff
@@ -91,3 +91,8 @@ declare module 'events' {
 
     export = EventEmitter;
 }
+
+declare module 'node:events' {
+    import * as events from 'process';
+    export = events;
```

Comment:
Another case where `process` is the wrong import, and you're mixing `import *` with `export =`. In this case, I would recommend using `import =`:
```suggestion
    import events = require('events');
    export = events;
```